src/main.rs

```diff
@@ -26,10 +26,10 @@ fn write(&self, string: &[u8]) -> Result<T, std::io::Error>;
         trait MsgWriter<'a, T, F, Z> {
             type WriterType : AnyWriter<'a, T, F>;
             /// Write a message somewhere.
-                /// A Result is returned for better error handling. Rust's approach is far superior
-            /// to the ridiculous try-catch blocks you usually see. Rust's way allows you to explicitly
+                /// A Result is returned for better error handling. Rust(🚀)'s approach is far superior
+            /// to the ridiculous try-catch blocks you usually see. Rust(🚀)'s way allows you to explicitly
                         /// name which error(s) can be returned (of course, this is unlikely to happen because
-                            /// Rust is so safe), and it's better than the way Java does it because the syntax isn't
+                            /// Rust(🚀) is so safe), and it's better than the way Java does it because the syntax isn't
```

Comment:
should there be an emoji for java too? :thinking:

Reply:
Java doesn't deserve it's own emoji.
Not even a 💩.
Moreover, the person reading may be confused of "What's Java? I thought I was reading Rust(🚀) source code. Now they are drawing our attention to Java?"

Reply:
fair